File: apps/web/src/code/python/basics/process_stop.py
from e2b import Sandbox

sandbox = Sandbox(template="base")

npm_init = sandbox.process.start("npm init -y")
npm_init.kill()  # $HighlightLine

# There will be no output because we immediately kill the `npm_init` process
print(npm_init.stdout)

sandbox.close()

File: apps/web/src/code/js/basics/process_stop.js
import { Sandbox } from 'e2b'

const sandbox = await Sandbox.create({
  template: 'base',
})

const npmInit = await sandbox.process.start({
  cmd: 'npm init -y',
})
await npmInit.kill() // $HighlightLine
// There will be no output because we immediately kill the `npm_init` process
console.log(npmInit.output.stdout)

await sandbox.close()
